src/view/gui/fMainWindow.cpp

```diff
@@ -5939,7 +5942,14 @@ void fMainWindow::openDicomImages(QString dir)
 
 void fMainWindow::ApplicationLIBRABatch()
 {
-  std::string scriptToCall = m_allNonNativeApps["libra"];
+  ApplicationDownloadManager* appDownloadMngr = new ApplicationDownloadManager();
```

Comment:
Same change as stated earlier: let's refrain from using raw pointers unless _absolutely_ needed. If raw pointers are needed, they should be smart pointers.